buildflags/BUILD.gn

```diff
@@ -20,6 +20,7 @@ buildflag_header("buildflags") {
     "ENABLE_ELECTRON_EXTENSIONS=$enable_electron_extensions",
     "ENABLE_BUILTIN_SPELLCHECKER=$enable_builtin_spellchecker",
     "ENABLE_PICTURE_IN_PICTURE=$enable_picture_in_picture",
+    "ENABLE_WIN_DARK_MODE=$enable_win_dark_mode",
```

Comment:
Can we call it something more specific to this feature.  i.e. `ENABLE_WIN_DARK_MODE_WINDOW_UI` or something

Reply:
Renamed in 42214684722c2aad1548ee539aa2bf65046d009a